pkg/rhcos/updatepayload/updatepayload.go

```diff
@@ -0,0 +1,157 @@
+// Package updatepayload extracts the RHCOS build version from an
```

Comment:
Also pulling the release image `might` require pullsecret...

Reply:
> Also pulling the release image might require pullsecret...

Yup.  I've filed containers/image#588 to give me a way to plug that in.

Reply:
> I've filed containers/image#588 to give me a way to plug that in.

This is taking a while in review, so I've added some local secret handling as part of 327976e5e -> 7c1ad4f6d.